frontend/ui/widget/keyvaluepage.lua

```diff
@@ -113,7 +113,8 @@ end
 local KeyValueItem = InputContainer:new{
     key = nil,
     value = nil,
-    cface = Font:getFace("cfont"),
+    cface = Font:getFace("cfont", 22),
```

Comment:
It shouldn't; that's done by getFace and if it weren't that's still where it should be done. ;-)